Add new language docs (#27)
exercism/v3: languages/csharp/README.md

# C&#35;

Welcome to the workspace for Exercism v3's C# track!

This area will contain everything needed to launch the C# track, including:

- The new exercises.
- An updated `config.json` file.
- Reference documentation that help explain C# concepts for which no appropriate online document could be found.
- C#-specific documentation for contributors.

> ## Prepartion Status

Reviewer: ```suggestion
## Preparation Status
```

> Before we publicise requesting contribution for this language, the following steps should be done.

Reviewer: I think we're using US English, aren't we? 

```suggestion
Before we publicize requesting contribution for this language, the following steps should be done.
```

> - [ ] [Listed out concepts](../../docs/determining-concepts.md)

Reviewer: ```suggestion
- [ ] [Listed out concepts](../../docs/maintainers/determining-concepts.md)
```

- [ ] [Added GitHub issues for 20 Concept Exercises](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
- [ ] Added documentation for contributors

## Readiness for Launch

> Before launch, we need all of the following parts build:

Reviewer: ```suggestion
Before launch, we need all of the following parts to be completed:
```

Some of the things listed are not really "built," such as updating the config.json.

### Track Structure

- [ ] Implemented 20+ Concept Exercises

> - [ ] [Updated `config.json`](https://github.com/iHiD/v3/blob/master/docs/maintainers/README.md#migrating-your-configjson-files)

Reviewer: ```suggestion
- [ ] [Updated `config.json`](../docs/maintainers/README.md#migrating-your-configjson-files)
```

>   - [ ] Added version key

Reviewer: ```suggestion
  - [ ] Added `version` key
```

  - [ ] Added online editor settings

>     - [ ] Added `indent_style`

Author: Yes, because we made add more keys, which will add then add to READMEs and I want people to be able to see clearly that a new requirement has been added.
